dev/tests/static/testsuite/Magento/Test/Legacy/_files/obsolete_classes.php

```diff
@@ -3350,6 +3350,8 @@
     ['Magento\Tools\Di\Compiler\Config\Reader', 'Magento\Setup\Module\Di\Compiler\Config\Reader'],
     ['Magento\Tools\Di\Compiler\Config\Writer\Filesystem', 'Magento\Setup\Module\Di\Compiler\Config\Writer\Filesystem'],
     ['Magento\Tools\Di\Compiler\Config\WriterInterface', 'Magento\Setup\Module\Di\Compiler\Config\WriterInterface'],
+    ['Magento\Setup\Module\Di\Compiler\Config\WriterInterface', 'Magento\Framework\App\ObjectManager\ConfigWriterInterface'],
```

Comment:
You shouldn't have added classes to obsoletes unless they are physically removed